src/app/api/s3-upload/route.ts

```diff
@@ -5,4 +5,11 @@ export const POST = route.configure({
   key(req, filename) {
     return sanitizeKey(filename).toLowerCase()
   },
+  accessKeyId: process.env.S3_UPLOAD_KEY,
+  secretAccessKey: process.env.S3_UPLOAD_SECRET,
+  bucket: process.env.S3_UPLOAD_BUCKET,
+  region: process.env.S3_UPLOAD_REGION,
```

Comment:
these are superfluous i think, only `endpoint` and `forcePathStyle` are needed if the rest are the default env vars used

Reply:
Thank you! I removed them as they were not needed indeed.